src/panels/lovelace/cards/hui-weather-forecast-card.ts

```diff
@@ -229,10 +229,14 @@ class HuiWeatherForecastCard extends LitElement implements LovelaceCard {
                   <div class="temp-attribute">
                     <div class="temp">
                       ${formatNumber(
-                        stateObj.attributes.temperature,
+                        stateObj.attributes.temperature!, // TODO: Fixme
```

Comment:
FIX ME! 😛

Reply:
Right, I need some help with this one. What should we do if the temperature is missing?

Reply:
Check if Temp Exist first before rendering the div

Reply:
OK, updated so the card is displayed like this if the temperature is missing:
![image](https://user-images.githubusercontent.com/14281572/175237236-c39c233c-bc2c-4c30-9e0d-6cb1e1d767c7.png)

Previously it would be displayed like this:
![image](https://user-images.githubusercontent.com/14281572/175237054-5c058dfb-631d-4f36-bd19-a7e747fcf217.png)

Reply:
Updated to render the div with a non-breaking space instead:
![image](https://user-images.githubusercontent.com/14281572/175249768-be941a59-8430-48c5-bf1c-dccf29e57406.png)